src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int index = Main.getRandom().nextInt(4);
```

Comment:
create constant from 4, for example, 4 -> `private static final int FIGURES_AMOUNT = 4;`